fdbserver/ClusterController.actor.cpp

```diff
@@ -3077,7 +2939,6 @@ ACTOR Future<Void> clusterControllerCore( ClusterControllerFullInterface interf,
 	state uint64_t step = 0;
 	state Future<ErrorOr<Void>> error = errorOr( actorCollection( self.addActor.getFuture() ) );
 
-	self.addActor.send( failureDetectionServer( self.id, &self, interf.clientInterface.failureMonitoring.getFuture() ) );
```

Comment:
Should `failureMonitoring` field be removed as well? And `FailureMonitoringRequest`?

Reply:
This is a good point. We cannot do that for 6.3 branch because it changes the interface's endpoints --> protocol change. 

We can do this in master though. 

I guess @vishesh can have another PR to remove the `failureMonitoring ` endpoint against master.

Reply:
Yes, I didn't remove the field because I didn't want to break protocol for 6.3. But yes, the plan was to create separate PR for master once this goes to 6.3 and subsequently master.